tests/test_utils/test_schemaview.py

```diff
@@ -22,6 +22,19 @@
 
 class SchemaViewTestCase(unittest.TestCase):
 
+    def test_schemaview_enums(self):
+        view = SchemaView(SCHEMA_NO_IMPORTS)
+        for en, e in view.all_enums().items():
+            if e.name == "Animals":
+                for pv, v in e.permissible_values.items():
+                    if pv == "CAT":
+                        assert view.permissible_value_ancestors(pv, e.name) == ['CAT']
```

Comment:
Consider using `TestCase` assertion methods instead of bare `assert`s! They provide much better diagnostic messages in the event of a failure. https://docs.python.org/3/library/unittest.html#assert-methods

Reply:
alright, I can clean up the inconsistent use for this class.  It looks like we have a mix of both styles.